orttraining/orttraining/test/graph/gradient_graph_builder_test.cc

```diff
@@ -97,6 +97,7 @@ static std::unique_ptr<TrainingSession> RunTrainingSessionWithChecks(
   RunOptions run_options;
   run_options.run_log_verbosity_level = so.session_log_verbosity_level;
   run_options.run_tag = so.session_logid;
+  run_options.training_mode = true;
```

Comment:
It should be true by default?

Reply:
maybe add ENABLE_TRAINING flag?

---
In reply to: [445680752](https://github.com/microsoft/onnxruntime/pull/4324#discussion_r445680752) [](ancestors = 445680752)

Reply:
Yea it'd be better to be guarded with Enable_training then and default to true.